Source/WebInspectorUI/UserInterface/Base/BrowserInspectorFrontendHost.js

```diff
@@ -226,13 +226,25 @@ if (!window.InspectorFrontendHost) {
 
         canSave(saveMode)
         {
-            return false;
+            // Support single file mode only.
+            return saveMode === WI.FileUtilities.SaveMode.SingleFile;
         }
 
         save(saveDatas, forceSaveAs)
         {
-            // FIXME: Create a Blob from the content, get an object URL, open it to trigger a download.
-            throw "unimplemented";
+            console.assert(saveDatas.length === 1, saveDatas);
+
+            let saveData = saveDatas[0];
+            let blob = WI.BlobUtilities.blobForContent(saveData.content, saveData.base64Encoded, "octet/stream");
+            let blobURL = window.URL.createObjectURL(blob);
+
+            let a = document.createElement("a");
+            a.href = blobURL;
+            a.style.display = "none";
+            // url looks like "web-inspector:///file.ext" use filename only
```

Comment:
Style: comments should be full sentences

Reply:
Fixed.